run/GCClassic/input_options.yml.templates/input_options.yml.fullchem

```diff
@@ -0,0 +1,426 @@
+### parsers will confuse this with a negative "no" value.
+
+#============================================================================
+# Simulation settings
+#============================================================================
+simulation:
+  name: ${RUNDIR_SIM_NAME}
+  start_date: [${RUNDIR_SIM_START_DATE}, ${RUNDIR_SIM_START_TIME}]
+  end_date: [${RUNDIR_SIM_END_DATE}, ${RUNDIR_SIM_END_TIME}]
+  root_data_dir: ${RUNDIR_DATA_ROOT}
+  met_field: ${RUNDIR_MET}
+  species_database_file: ./species_database.yml
+  debug_printout: false
+  use_gcclassic_timers: ${RUNDIR_USE_GCCLASSIC_TIMERS}
+
+#============================================================================
+# Grid settings
+#============================================================================
+grid:
+  resolution: ${RUNDIR_GRID_RES_LONG}
+  number_of_levels: ${RUNDIR_GRID_NLEV}
+  longitude:
+    range: ${RUNDIR_GRID_LON_RANGE}
+    center_at_180: ${RUNDIR_CENTER_LON_180}
+  latitude:
+    range: ${RUNDIR_GRID_LAT_RANGE}
+    half_size_polar_boxes: ${RUNDIR_GRID_HALF_POLAR}
+  nested_grid_simulation:
+    activate: ${RUNDIR_GRID_NESTED_SIM}
+    buffer_zone_NSEW: ${RUNDIR_GRID_BUFFER_ZONE}
+
+#============================================================================
+# Timesteps settings
+#============================================================================
+timesteps:
+  transport_timestep_in_s: ${RUNDIR_TRANSPORT_TS}
+  chemistry_timestep_in_s: ${RUNDIR_CHEMISTRY_TS}
+  radiation_timestep_in_s: 10800
+
+#============================================================================
+# Settings for GEOS-Chem operations
+#============================================================================
+operations:
+
+  chemistry:
+    activate: true
+    linear_chemistry_aloft:
+      activate: true
+      use_linoz_for_O3: true
+    active_strat_H2O:
+      activate: true
+      use_static_bnd_cond: true
+    gamma_HO2: 0.2
+
+  convection:
+    activate: true
+
+  dry_deposition:
+    activate: true
+    CO2_effect:
+      activate: false
+      CO2_level: 600.0
+      reference_CO2_level: 380.0
+    diag_alt_above_sfc_in_m: 10
+
+  pbl_mixing:
+     activate: true
+     use_non_local_pbl: ${RUNDIR_USE_NLPBL}
+
+  photolysis:
+    input_directory: ${RUNDIR_DATA_ROOT}/CHEM_INPUTS/FAST_JX/v2021-10/
+    overhead_O3:
+      use_online_O3_from_model: ${RUNDIR_USE_ONLINE_O3}
+      use_column_O3_from_met: true
+      use_TOMS_SBUV_O3: false
+    photolyze_nitrate_aerosol:
+      activate: false
+      NITs_Jscale_JHNO3: 0.0
+      NIT_Jscale_JHNO2: 0.0
+      percent_channel_A_HONO: 66.667
+      percent_channel_B_NO2: 33.333
+
+  rrtmg:
+    activate: false
+    aod_wavelengths_in_nm:
+      - 550
+    longwave_fluxes: ${RUNDIR_RRTMG_OPTS}
+    shortwave_fluxes: ${RUNDIR_RRTMG_OPTS}
+    clear_sky_flux: ${RUNDIR_RRTMG_OPTS}
+    all_sky_flux: ${RUNDIR_RRTMG_OPTS}
+
+  transport:
+    activate: true
```

Comment:
This section is tough since it lists advected species but activate will be false for GCHP. That makes it seem the details in the section are not applicable in GCHP. Would it be possible to omit GC-Classic-only lines in the GCHP yml, such as transport activate? Alternatively, maybe we can figure out how best to indicate which lines are GC-Classic only. And yet another option is move the advection species to their own section and label this one gcclassic_transport.

Reply:
We should be able to omit fields that do not apply to GCHP -- we don't have to include them in the GCHP rundir geoschem_config.yml file. Fields that are not found are set to missing values (e.g. false, or -999, etc.)

Also I think we could do something like this:

```yaml
  transport:
    gcclassic_tpcore:                 # GEOS-Chem Classic only
      activate: true                  # GEOS-Chem Classic only
      fill_negative_values: true      # GEOS-Chem Classic only
      iord_jord_kord: [3, 3, 7]       # GEOS-Chem Classic only
    transported_species:
      - ACET
      - ACTA
      ..etc...
    passive_species:
      PassiveTracer:
        long_name: Passive_tracer_for_mass_conservation_evaluation
        mol_wt_in_g: 1.0
        lifetime_in_s: -1
        default_bkg_conc_in_vv: 1.0e-7
      SF6Tracer:
        long_name: Sulfur_hexafluoride
        mol_wt_in_g: 146.0
        lifetime_in_s: -1
        default_bkg_conc_in_vv: 1.0e-20
      CH3ITracer:
        long_name: Methyl_iodide
        mol_wt_in_g: 142.0
        lifetime_in_s: 4.32e5
        default_bkg_conc_in_vv: 1.0e-20
```

Reply:
this is just an example to show how you can add passive species to any simulation if you want.

Reply:
Now fixed in a6cc6617a6ecb9597e8d39ce7ceff0eb880739e4 and 973c2266ccf64e8d587d1fa705eef60ba2896056